examples/apa102/itsybitsy-m0/main.go

```diff
@@ -47,7 +56,7 @@ func main() {
 			if !brightening {
 				brightness = 0xFFFF - brightness
 			}
-			led.Set(brightness)
+			pwm.Set(channelLED, uint32(brightness)/0xFFFF)
```

Comment:
If you want to set the PWM to a certain fraction, you need to use the `Top()` method. For example:

```suggestion
			pwm.Set(channelLED, pwm.Top() * 256 / brightness)
```

This also requires removing `uint16(i) << 8`: there is no need for increasing the range here.

Reply:
I think you meant `pwm.Set(channelLED, pwm.Top()*brightness/256)`.

Anyhow changes made and force pushed to branch.